lib/cisco_node_utils/overlay_global.rb

```diff
@@ -127,8 +127,7 @@ def default_dup_host_mac_detection_timeout
     def anycast_gateway_mac
       return nil unless Feature.nv_overlay_evpn_enabled?
       mac = config_get('overlay_global', 'anycast_gateway_mac')
-      # This value gets 0-padded when nvgened, so we need to convert it.
-      Utils.zero_pad_macaddr(mac).nil? ? default_anycast_gateway_mac : mac
+      mac.nil? || mac.empty? ? default_anycast_gateway_mac : mac.downcase
```

Comment:
Yes, from the PR description:
*This change will also be made to puppet in a subsequent commit to downcase the manifest data.*